LocalSettings.php

```diff
@@ -951,6 +951,9 @@
 	'wmgUseEventLogging' => [
 		'default' => false,
 	],
+	'$wgExtraSignatureNamespaces' => [
+		'nonciclopediawiki' => [NS_HELP, NS_PROJECT],
```

Comment:
needs to be [ NS_HELP, NS_PROJECT ],

Also need to set a default 'default' => [],